secure/key-manager/src/tests.rs

```diff
@@ -15,8 +15,8 @@ use diem_crypto::{ed25519::Ed25519PrivateKey, HashValue, PrivateKey, Uniform};
 use diem_global_constants::{
     CONSENSUS_KEY, OPERATOR_ACCOUNT, OPERATOR_KEY, OWNER_ACCOUNT, OWNER_KEY,
 };
-use diem_secure_storage::{InMemoryStorageInternal, KVStorage};
-use diem_secure_time::{MockTimeService, TimeService};
+use diem_secure_storage::{InMemoryStorage, KVStorage};
+use diem_time_service::{MockTimeService, TimeServiceTrait};
```

Comment:
Can we document (maybe I missed it), that the `TimeServiceTrait` needs to be imported to use the `TimeService` functions directly?